tox.ini

```diff
@@ -11,6 +11,11 @@ envlist =
     pypy3-test-resource-detector-container
     lint-resource-detector-container
 
+    ; opentelemetry-resource-detector-azure
+    py3{8,9,10,11,12}-test-resource-detector-azure
+    pypy3-test-resource-detector-azure
+    lint-resource-detector-azure
```

Comment:
I suppose we should add the steps to lint as well right?

Reply:
Ah, do you mean to add the commands right?